raiden-ts/CHANGELOG.md

```diff
@@ -19,8 +19,10 @@
 
 ### Changed
 - [#1462] Refactor state schema and types to be simpler and safer
+- [#1480] Update profile's caps on config.caps change and react on peers updates
 
 [#1462]: https://github.com/raiden-network/light-client/issues/1462
+[#1480]: https://github.com/raiden-network/light-client/pull/1480
```

Comment:
We made last weeks release today, so this would need to go into a new unreleased section

Reply:
Yes, this PR was done before the release